index.rst

```diff
@@ -3,21 +3,79 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Next-generation file formats (NGFF)
-===================================
+Next-Generation File Formats (NGFF) + OME-Zarr
+===============================================
 
-NGFF is an initiative by the bioimaging community to develop imaging format specifications to
-address issues of scalability and interoperability.
-Please see the :doc:`about/index` section for an introduction.
-The OME-Zarr specification is detailed under :doc:`specifications/index`.
-Various Image viewers and other software for working with OME-Zarr data
-are listed on the :doc:`tools/index` page.
-Sample OME-Zarr datasets provided by the community can be found under :doc:`data/index`.
+**Contents**
+
+* NGFF vs OME-Zarr, what is the difference?
+* What is an OME-Zarr?
+* Why would I use OME-Zarr?
+* When would I not use OME-Zarr?
+* How do I use OME-Zarr?
+* Who is using OME-Zarr?
+* ...have other questions? `Check out the FAQ </help-desk/index.html>`_
```

Comment:
I would argue that we don't need a table of contents at the top (esp. since it's not clickable) since there's a nice scrolling and clickable table of contents on the right side of the page. Instead, it might be nice to have a very short welcoming blurb here. Would have to move the "other questions, check out the FAQ" lie to the end of the page